src/i18n/en.json

```
@@ -52,6 +52,14 @@
 
     "transaction.debug.description": "DEBUG: context: {context} methodName: {methodName}",
 
+    "transaction.group.deposit.title": "Activate tokens",
+    "transaction.group.deposit.description": "Activate tokens",
+    "transaction.TokenLockingClient.deposit.title": "Activate tokens",
+    "transaction.group.withdraw.title": "Withdraw tokens",
+    "transaction.group.withdraw.description": "Withdraw tokens",
+    "transaction.TokenLockingClient.wthdraw.description": "Withdraw tokens",
+    "transaction.TokenLockingClient.withdraw.title": "Withdraw tokens",
+    "transaction.TokenClient.approve.title": "Approve amount of tokens",
```

As above, this needs a description message descriptor counterpart

```suggestion
    "transaction.TokenClient.approve.title": "Approve amount of tokens",
    "transaction.TokenClient.approve.description": "Approve amount of tokens",
```